.goreleaser.yml

```diff
@@ -21,6 +21,26 @@ builds:
       - linux
     goarch:
       - amd64
+  - id: confluent-alpine-amd64-rpm
```

Comment:
```suggestion
  - id: confluent-alpine-amd64-disableupdates
```

Reply:
Why are we updating the alpine build instead of the regular linux build? I thought alpine was just for Docker.

Reply:
Oh, right. This is not what we want. Well this actually simplifies it further, since I can just use the homebrew builds for rpm and deb. I which case I'll just rename `homebrew` -> `disableupdates` and remove this block.

I'm sort of confused on why I was able to run `confluent` after installing it with yum on our centos image... I guess musl is installed there.

Reply:
> I can just use the homebrew builds for rpm and deb

:+1: